scanner/src/components/DataProduct/index.svelte

```diff
@@ -83,13 +85,13 @@
       <button class="button unsupported" disabled> Unsupported </button>
     {/if}
   </div>
-  <div class="subtitle" class:divider={open}>{productMetadata.description}</div>
   {#if loadedData}
+    <div class="subtitle" class:divider={open}>{productMetadata.description}</div>
```

Comment:
just curious why you can't use Subtitle + Divider there instead. I didn't compare closely, but their styles look kinda similar to styles that you've defined in this file for .subtitle and .divider